app/components/notification-list-row/template.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="flex-container">
+    <div class="notification-title">New pending submissions</div>
+    {{#bs-dropdown as |dd|}}
+        {{#dd.button class="dropdown"}} {{t currentSettingLabel}} <span class="fa fa-caret-down p-l-xs"></span>{{/dd.button}}
+        {{#dd.menu classNames="notification-menu" as |menu|}}
+            {{#each notificationOptions as |notificationOption| }}
+                {{#menu.item}}
+                    <button class="btn" onclick={{perform updateFrequency notificationOption.value}}>
```

Comment:
Can we give this button a `min-width` of 86px so that the button size doesn't change when selecting different options. It will matter more when we add other settings to the page but it's better to take care of it now.

Then to make it so the spacing is dynamic between the label and the caret add:
```
.notification-menu {
  button {
    display: flex;
    justify-content: space-between;
  }
}
```

Reply:
I believe you are referring to the button 4 lines above. (See the previous comment.) If so, I've added a new css file for the `notification-list-row` component to address the width and spacing issue.